front/app/containers/MainHeader/Components/NavbarContent/MobileNavbarContent/FullMobileNavMenu.tsx

```diff
@@ -119,18 +106,18 @@ const FullMobileNavMenu = ({
   mobileNavbarRef,
   onClose,
   isFullMenuOpened,
-  intl: { formatMessage },
-}: Props & WrappedComponentProps) => {
+}: Props) => {
   const { data: navbarItems } = useNavbarItems();
   const pageSlugById = useCustomPageSlugById();
   const localize = useLocalize();
+  const { formatMessage } = useIntl();
 
   if (isNilOrError(navbarItems) || isNilOrError(pageSlugById)) return null;
 
   const navbarItemPropsArray = getNavbarItemPropsArray(
     navbarItems.data,
     pageSlugById
-  ).filter(({ linkTo }) => linkTo !== '/projects'); // /projects is already in the 'outer' fixed menu
+  );
```

Comment:
We do want to include "All projects" now in this list, so I've removed the filter.